recipe/run_test.py

```diff
@@ -73,8 +73,10 @@
 
 extra_argv = []
 if os.getenv("CI") != "travis":
```

Comment:
I would invoke pytest directly, instead of going through scipy, their wrapper is rather custom.

But anyway, you might want to build in parallel as well:

See
https://github.com/conda-forge/scikit-learn-feedstock/blob/master/recipe/meta.yaml#L20

Reply:
It ran for 35 mins and timedout???

Add pytest timeout as well, with a 5 min timeout on each test.

All this stuff is easier to do with pytest directly, instead of scipy (because things are more portable between tests).

IMO

Reply:
"The job exceeded the maximum log length, and has been terminated."

Reply:
ohhh, yeah, make thigns less verbose.